src/view.rs

```diff
@@ -197,6 +197,11 @@ impl View {
         app.set_after_command(self.after_command.clone());
 
         // set mouse events
+        // Windows mouse capture implemention requires EnableMouseCapture be invoked before DisableMouseCatpure can be used
+        // https://github.com/crossterm-rs/crossterm/issues/660
+        if cfg!(target_os = "windows") {
+            execute!(terminal.backend_mut(), EnableMouseCapture)?;
+        }
         app.set_mouse_events(self.mouse_events);
```

Comment:
The idea is: if on windows, first enable mouse capture; then immediately set mouse events to whatever was supplied via clap argument. `hwatch`'s clap argument for mouse capture defaults to false so this change preserves the current functionality on macOS and *nixes, while fixing this Windows-specific issue.